blueflood-http/src/integration-test/java/com/rackspacecloud/blueflood/outputs/handlers/HttpMultiRollupsQueryHandlerIntegrationTest.java

```diff
@@ -51,54 +64,64 @@ public void testMultiplotQuery() throws Exception {
         JsonParser jsonParser = new JsonParser();
         JsonObject responseObject = jsonParser.parse(responseContent).getAsJsonObject();
 
-        String expectedResponse = "{\n" +
-                "  \"metrics\": [\n" +
-                "    {\n" +
-                "      \"unit\": \"unknown\",\n" +
-                "      \"metric\": \"3333333.G1s\",\n" +
-                "      \"data\": [\n" +
-                "        {\n" +
-                "          \"numPoints\": 1,\n" +
-                "          \"timestamp\": 1382400000,\n" +
-                "          \"latest\": 397\n" +
-                "        }\n" +
-                "      ],\n" +
-                "      \"type\": \"number\"\n" +
-                "    },\n" +
-                "    {\n" +
-                "      \"unit\": \"unknown\",\n" +
-                "      \"metric\": \"3333333.G10s\",\n" +
-                "      \"data\": [\n" +
-                "        {\n" +
-                "          \"numPoints\": 1,\n" +
-                "          \"timestamp\": 1382400000,\n" +
-                "          \"latest\": 56\n" +
-                "        }\n" +
-                "      ],\n" +
-                "      \"type\": \"number\"\n" +
-                "    }\n" +
-                "  ]\n" +
-                "}";
-        JsonObject expectedObject = jsonParser.parse(expectedResponse).getAsJsonObject();
-
-        Assert.assertEquals(expectedObject, responseObject);
-        EntityUtils.consume(query_response.getEntity());
+        JsonArray metrics = responseObject.getAsJsonArray( "metrics" );
```

Comment:
Since we now need to post events with a current timestamp, the straight string compare wasn't sufficient.  Also, the order of the metrics can change due to the prefixed (or will be postfixed) metric names.